README.md

```diff
@@ -184,6 +184,25 @@ Samila is a generative art generator written in Python, Samila let's you create
 
```

Comment:
We should update this section. 
Something like:

7. List (example: `color=["black", "#fffeef",...]`)

⚠️ **List** mode is only available for color
⚠️ In **List** mode, the length of this list must be equal to the lengths of `data1` and `data2`.